.github/workflows/upgrade-python-requirements.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: setup testeng-ci
         run: |
-          git clone https://github.com/edx/testeng-ci.git
+          git clone https://github.com/openedx/testeng-ci.git
```

Comment:
I think this repo is maintained by the folks at 2U, according to [this](https://github.com/openedx/openedx-events/pull/97#discussion_r953009436) comment

Reply:
All right, let me know! :heart: